source/_integrations/purpleair.markdown

```diff
@@ -29,6 +29,19 @@ Remove Sensor:
   description: Untrack a sensor.
 {% endconfiguration_basic %}
 
+## Creating an API key
+
+To add this integration you will need a (free) Purple Air API Key.  Detailed instructions are at [https://community.purpleair.com/t/creating-api-keys/3951]([https://community.purpleair.com/t/creating-api-keys/3951]) but in summary you:
```

Comment:
It might be worth noting that while a new account comes with 1M free points, after that users will need to buy additional points to continue to use the API. As currently written, the current plugin uses ~30K points/day, so the 1M points will last about a month before the user runs into API errors unless they buy more points. See #93428 for details.